dataland-e2etests/src/test/kotlin/org/dataland/e2etests/EuTaxonomyControllerTest.kt

```diff
@@ -3,21 +3,28 @@ package org.dataland.e2etests
 import org.dataland.datalandbackend.openApiClient.api.CompanyDataControllerApi
 import org.dataland.datalandbackend.openApiClient.api.EuTaxonomyDataControllerApi
 import org.dataland.datalandbackend.openApiClient.model.CompanyAssociatedDataEuTaxonomyData
-import org.dataland.datalandbackend.openApiClient.model.PostCompanyRequestBody
+import org.dataland.datalandbackend.openApiClient.model.CompanyInformation
 import org.junit.jupiter.api.Assertions.assertEquals
 import org.junit.jupiter.api.Test
+import java.math.BigDecimal
+import java.time.LocalDate
 
 class EuTaxonomyControllerTest {
-    private val basePathToBackendViaProxy = "http://proxy:80/api"
-    private val companyDataControllerApi = CompanyDataControllerApi(basePathToBackendViaProxy)
-    private val euTaxonomyDataControllerApi = EuTaxonomyDataControllerApi(basePathToBackendViaProxy)
+    private val basePathToDatalandProxy = "http://proxy:80/api"
```

Comment:
Bin mir nicht sicher ob der Name hier so gut ist wie wäre es mit "basePathToDatalandApi" das beschreibt meiner Meinung nach besser was es ist. Das die API hinter dem proxy liegt muss in der Variable nicht stehen denke ich.

Reply:
Wurde in eine constante umgelegt daher obsoleter Kommentar